### ISSUE TITLE
level= 0;1 (etc) are changed to level=0 when POI edited

### ISSUE THREAD

**Issue by RedAuburn** [organicmaps/organicmaps#9308]
looks like OM doesn't handle multiple level values well.

there are 108,893 `level` tags with `;`, of 1,659,286 (6.56%)

![image](https://github.com/user-attachments/assets/ff146e86-8556-4e7d-b1f7-3dcf9411e078)

https://osmcha.org/changesets/156777154

would be partially fixed by https://github.com/organicmaps/organicmaps/pull/9242

**Comment by map-per** [comment]
Level is also displayed wrong in the Place Page (e.g. for elevators with several levels like [this](https://www.openstreetmap.org/node/3159190244) one). So this is probably a bug in the Map Generator.

**Comment by RedAuburn** [comment]
> But even with the bug in the Map Generator OM shouldn't have changed OSM data. There is no UI for editing level, so this change was entirely made up by OM.

yep, i made the edit and it was only to add the website. This means that any object with this type of level=* tagging that has ever been touched by an OM user has been wrongly changed 🥲

**Comment by map-per** [comment]
Looks like the problem is here:
https://github.com/organicmaps/organicmaps/blob/a570d855f7c974ef123f65b3475f154ff1f5e061/generator/osm2meta.cpp#L239-L248

The generator converts level values to double and then back to string. That's why only the first number survives the process. E.g. `level=-1;0;1` becomes `level=-1`
